cql/ManageSpecialPopulation.cql

```diff
@@ -302,7 +302,7 @@ define RecommendationForPatientsYoungerThan25:
         date: Collate.DateOfMostRecentReport + 1 year,
```

Comment:
Heading 3.1b in `ManageSpecialPopulation.cql` could also benefit from an update.

Reply:
Missed that one. Now refers to "within 1 year from date of most recent test"

Reply:
Same comment as I had above, about the fact that changing "in" to "within" slightly changes the meaning of the recommendation. I will keep this thread resolved, so as not to be duplicative, and will keep the other one open for us to discuss there.